docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,217 @@
+# that recently arrived)
+#
+# Check out the :doc:`when should you use </getting-started/when_should_you_use>`
+# deepchecks guide for some more info about the existing suites and when to use them.
+
+from deepchecks.tabular.suites import train_test_validation
+
+validation_suite = train_test_validation()
+suite_result = validation_suite.run(train_ds, test_ds)
+# Note: the result can be saved as html using suite_result.save_as_html()
+# or exported to json using suite_result.to_json()
+suite_result
+
+#%%
+# We can see that we have a problem in the way we've split our data!
+# We've mixed up data from two years, causing a leakage of future data
+# in the training dataset.
+# Let's fix this.
+# 
+# Fix Data
+# ^^^^^^^^^^
+
+dt_col = data[datetime_name]
+train_df = data[dt_col.dt.year.isin([2017]) & dt_col.dt.month.isin([6,7,8])]
+test_df = data[dt_col.dt.year.isin([2018]) & dt_col.dt.month.isin([6,7,8])]
+
+#%%
+
+from deepchecks.tabular import Dataset
+
+# Create the new Datasets
+train_ds = Dataset(train_df, **columns_metadata)
+test_ds = Dataset(test_df, **columns_metadata)
+
+#%%
+#
+# Re-run Validation Suite
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+
+suite_result = validation_suite.run(train_ds, test_ds)
+suite_result
+
+#%%
+#
+# Ok, the date leakage doesn't exist anymore!
+#
+# However, we can see that we have a multivariate drift in the current split, 
+# detected by the :doc:`</checks_gallery/tabular/train_test_validation/plot_whole_dataset_drift>` check.
+# The drift is detected mainly by a combination of features representing the loan's interest rate (``int_rate``) 
+# and its grade (``sub_grade``).
+#
+# We can consider examining other sampling techniques (e.g. using only data from the same year), 
+# ideally achieving one in which the training data's
+# univariate and multivariate distribution is 
+# similar to the data on which the model will run (test / production data).qgjjj
+#
+# If we are planning on training a model with these splits, this drift is worth understanding 
+# (do we expect this kind of drift in the model's production environment? can we do something about it?).
+# Otherwise, we can consider sampling or splitting the data differently, and using deepchecks to validate it.
+# For more details about drift, see the :doc:`</user-guide/general/drift_guide>`.
+
+
+
+#%%
+# Run a Single Check
+# -------------------
+#
+# We can run a single check on a dataset, and see the results.
+
+# If we want to run only that check (possible with or without condition)
+from deepchecks.tabular.checks import WholeDatasetDrift
+
+check_with_condition = WholeDatasetDrift().add_condition_overall_drift_value_less_than(0.4)
+# check = WholeDatasetDrift()
+dataset_drift_result = check_with_condition.run(train_ds, test_ds)
+
+#%%
+# We can also inspect and use the result's value:
+
+dataset_drift_result.value
+
+#%%
+# and see if the conditions have passed
+dataset_drift_result.passed_conditions()
+
+#%%
+# Create a Custom Suite
```

Comment:
Not sure why this is here either. Shouldn't we just link to the custom suite guide?

Reply:
answered also Gabbay with same question (but basically - relates to encouraging the basic usability etc. in a single tutorial without needing to navigate etc.)